packages/config/__tests__/__resources__/ProfInfoApp_user_and_team_config_proj/ProfInfoApp.schema.json

```diff
@@ -0,0 +1,186 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$version": 2,
```

Comment:
This schema will need to be updated in #751. Nothing to change in this PR, just a note for my own future reference 🙂